pages/index.js

```diff
@@ -10,6 +10,7 @@ export default function home({ data }) {
       <h3>(This top page is for humans)</h3>
       <p>Plugin index: <Link href={`/api/plugins`}><a>/api/plugins</a></Link></p>
       <p>Summary info: <Link href={`/api/extended_summary`}><a>/api/extended_summary</a></Link></p>
+      <p>PyPI information: see PyPI info for each plugin at /api/conda/pypi_name</p>
```

Comment:
Is this URL correct? Should it be `/api/pypi/pypi_name`?

Reply:
Better eyes than mine - I think you're right!